### PR TITLE
 Standardize integration testing

### PR DESCRIPTION
**Pull request by kopachevsky**
Fixes https://github.com/GoogleCloudPlatform/terraform-google-sql-db/issues/42
- added testing infra bash files
- renamed project to project_id param, like in other modules
- removed credentials path as Terraform variable in respect to ENV variable

### CODE REVIEW

**Comment by morgante** [comment]
@kopachevsky Please drop the "renamed project to project_id param, like in other modules"

It's a breaking change which is unnecessary and out of scope IMO.

**Comment by kopachevsky** [comment]
@morgante ok

File: test/fixtures/shared/terraform.tfvars [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#################################################################
+#   PLEASE FILL THE VARIABLES WITH VALID VALUES FOR TESTING     #
+#   DO NOT REMOVE ANY OF THE VARIABLES                          #
+#################################################################
+
+## These values you *MUST* modify to match your environment
```
[the remark this answers is not in the document]
Comment (kopachevsky) [reply test/fixtures/shared/terraform.tfvars]: changed to 

> ## These values are default for integration tests, you might change values for your environment if needed

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+## [Unreleased]
+
+### Added
+
+
+
+### Changed
+
+- Standardized integration testing [#48]
```
[the remark this answers is not in the document]
Comment (kopachevsky) [reply CHANGELOG.md]: removed change line

**Pull request merged**